pom.xml

```diff
@@ -963,4 +971,37 @@
     </snapshotRepository>
   </distributionManagement>
 
+  <profiles>
+    <!-- Spark version profile -->
+    <profile>
+      <id>spark-1.6</id>
+      <activation>
+        <property>
+          <name>!spark-2.0</name>
+        </property>
+      </activation>
+      <properties>
+        <spark.version>1.6.2</spark.version>
+        <py4j.version>0.9</py4j.version>
+        <netty.version>4.0.23.Final</netty.version>
+        <json4s.version>3.2.10</json4s.version>
+      </properties>
+    </profile>
+
+    <profile>
+      <id>spark-2.0</id>
```

Comment:
You didn't answer my question about the need for this profile. What is it used for? It seems all the 1.6 vs. 2.0 differences are being handled via reflection.